pkg/cli/advisory_guide.go

```diff
@@ -295,16 +511,100 @@ var (
 	sectionDivider = wrapped.Repeat("—")
 
 	welcomeMessage = func() string {
-		return fmt.Sprintf(welcomeMessageFormat, styleBold.Render("ctrl+C"))
+		return fmt.Sprintf(welcomeMessageFormat, styles.Bold().Render("ctrl+C"))
 	}()
 
 	errorForNotADistroDirectory = func(cwd string) string {
 		return fmt.Sprintf(
 			notADistroDirectoryMessageFormat,
-			styleBold.Render(cwd),
-			styleBold.Copy().Italic(true).Render("is"),
+			styles.Bold().Render(cwd),
+			styles.Bold().Copy().Italic(true).Render("is"),
 		)
 	}
 
 	styleBoldItalic = lipgloss.NewStyle().Bold(true).Italic(true)
 )
+
+var (
+	questionIsThisAFalsePositive = questions.Question[advisory.Request]{
+		Text: "Any obvious sign that this is a false positive?",
+		Choices: []questions.Choice[advisory.Request]{
+			{
+				Text: "No",
+				Choose: func(req advisory.Request) (updated advisory.Request, next *questions.Question[advisory.Request]) {
+					return req, &questionIsPackageSupported
+				},
+			},
+			{
+				Text: "Yes",
+				Choose: func(req advisory.Request) (updated advisory.Request, next *questions.Question[advisory.Request]) {
+					e := v2.Event{
+						Timestamp: v2.Now(),
+						Type:      v2.EventTypeFalsePositiveDetermination,
+					}
+
+					req.Event = e
+					return req, &questionWhyFalsePositive
+				},
+			},
+		},
+	}
+
+	questionWhyFalsePositive = questions.Question[advisory.Request]{
+		Text: "Why is this a false positive?",
```

Comment:
**Question**: where should it go ? `I ran `-govulncheck` and tells me this is a false positive...` Do we need to add this case ?

Reply:
Yes, the questions aren't completed yet, but we can iterate on those next...

> What's missing
> 
> To be useful, we need to iterate on the interview questions and answers to the point where they collectively cover an acceptable portion of cases that engineers encounter when triaging vulnerabilities in newly built APKs.

Reply:
sgtm